go2ts_test.go

```diff
@@ -49,9 +49,12 @@ func TestRender_ComplexStruct_Success(t *testing.T) {
 		OptionalOther         *OtherStruct                 `json:"oe,omitempty"`
 		Data                  Data                         `json:"d"`
 		DataPtr               *Data                        `json:"dp"`
-		MapStringSlice        map[string][]*string         `json:"mss"`
+		MapStringSlice        map[string][]string          `json:"mss"`
```

Comment:
In another PR we should remove all the json: annotations except just a couple to show that it works. the 'mss', 'mosp' renaming just makes it harder to match up the resulting TypeScript with the Go code.

Reply:
Great idea!